tests/base.py

```diff
@@ -26,7 +25,7 @@ def _head_request(self, url, user_agent=_user_agent_firefox,
         except requests.RequestException as e:
             request_url = self._build_request_url(url, params)
 
-            Assert.fail('Failing URL: %s.\nError message: %s' % (request_url, e))
+            assert False, 'Failing URL: %s.\nError message: %s' % (request_url, e)
```

Comment:
We should use `raise AssertionError('reason')` wherever we previously used `Assert.fail('reason')`.